extensions/2.0/Vendor/EXT_texture_avif/README.md

```diff
@@ -0,0 +1,109 @@
+# EXT_texture_avif
+
+## Contributors
+
+- Omar Shehata, Cesium, [@OmarShehata](https://github.com/OmarShehata)
+- Pascal Massimino, Google, [@Skal65535](https://github.com/skal65535)
+
+## Status
+
+Complete
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension allows glTF models to use AVIF as a valid image format. A client that does not implement this extension can ignore the provided AVIF image and continue to rely on the PNG and JPG textures available in the base specification. Defining a fallback texture is optional. The [best practices](#best-practices) section describes the intended use case of this extension and the expected behavior when using it without a fallback texture.
+
+AVIF is an image format developed by Netflix that uses the AV1 VideoCodec [specification](https://aomediacodec.github.io/av1-avif/).
+The format has gained popularity and you can see in what browsers it is supported here [caniuse](https://caniuse.com/avif)
+
+## glTF Schema Updates
+
+The `EXT_texture_avif` extension is added to the `textures` node and specifies a `source` property that points to the index of the `images` node which in turn points to the AVIF image.
+
+The following glTF will load `image.avif` in clients that support this extension, and otherwise fall back to `image.png`.
+
+```
+"textures": [
+    {
+        "source": 0,
+        "extensions": {
+            "EXT_texture_avif": {
+                "source": 1
+            }
+        }
+    }
+],
+"images": [
+    {
+        "uri": "image.png"
+    },
+    {
+        "uri": "image.avif"
+    }
+]
+```
+
+When used in the glTF Binary (.glb) format the `images` node that points to the AVIF image uses the `mimeType` value of `image/avif`.
+
+```
+"textures": [
+    {
+        "source": 0,
+        "extensions": {
+            "EXT_texture_avif": {
+                "source": 1
+            }
+        }
+    }
+],
+"images": [
+    {
+        "mimeType": "image/png",
+        "bufferView": 1
+    },
+    {
+        "mimeType": "image/avif",
+        "bufferView": 2
+    }
+]
+```
+
+### JSON Schema
+
+[glTF.EXT_texture_avif.schema.json](schema/glTF.EXT_texture_avif.schema.json)
+
+## Best Practices
+
+Since AVIF is not as widely supported as JPEG and PNG, it is recommended to use this extension only when transmission size is a significant bottleneck. For example, in geospatial applications the total texture payload can range from gigabytes to terabytes of data. In those situations, AVIF textures can be used without a fallback to improve storage and transmission.
+
+When a fallback image is defined, this extension should not be present in `extensionsRequired`. This will allow all clients to render the glTF correctly, and those that support this extension can request the optimized AVIF version of the textures.
+
```

Comment:
Given the flexibility of the format (auxiliary images, sequences, etc.), the extension spec must define an appropriate scope that is compatible with glTF texture usage.

Reply:
The core specification already defines color management. Other extra features that could be present in AVIF files should be addressed here.

Reply:
I added a part about this only being a replacement for jpeg / png.

I added support for the extension in gltf-pipeline today
https://github.com/leon/gltf-pipeline/tree/avif

an .avif file that is an image starts with ftypavif
whilst image sequences has another byte sequence,
I only added support for the image version for now.